websites/S/Speedtest/dist/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Nevylish",
+    "id": "343847696483745794"
+  },
+  "url": "www.speedtest.net",
+  "description": {
+    "de": "Der umfassende Breitband-Geschwindigkeitstest",
+    "en": "The Global Broadband Speed Test",
+    "es": "La prueba de velocidad de banda ancha global",
+    "fr": "Le test de vitesse de connexion global"
+  },
+  "service": "Speedtest",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/MOxmNLG.png",
+  "thumbnail": "https://i.imgur.com/JXIDlOL.png",
```

Comment:
While it's nice that the thumbnail is PreMiD Telecom and PreMiD.app as the URL, I'd just suggest changing it to different values (i.e a massive Telecom Provider), and then for the I.P just put 192.168.0.1 or 127.0.0.1 or localhost, just something not your IP obviously for privacy concerns.